File: app/src/main/java/com/tejasbhitle/gallery/util/MediaFileFilter.java
package com.tejasbhitle.gallery.util;

import java.io.File;
import java.io.FileFilter;

public class MediaFileFilter implements FileFilter {

    private final String[] okFileExtensions = new String[]
            { "jpg", "jpeg", "png", "gif","mp4", "avi", "mov" };

    @Override
    public boolean accept(File file) {
        for (String extension : okFileExtensions) {
            if (file.getName().toLowerCase().endsWith(extension)) {
                return true;
            }
        }
        return false;
    }
}

File: app/src/main/java/com/tejasbhitle/gallery/util/MediaFileFilter.kt
package com.tejasbhitle.gallery.util

import java.io.File
import java.io.FileFilter

class MediaFileFilter : FileFilter {

    private val okFileExtensions = arrayOf("jpg", "jpeg", "png", "gif", "mp4", "avi", "mov")

    override fun accept(file: File): Boolean {
        for (extension in okFileExtensions) {
            if (file.name.toLowerCase().endsWith(extension)) {
                return true
            }
        }
        return false
    }
}
